<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>connection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="address_comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="connect_pushButton">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnect_pushButton">
        <property name="text">
         <string>disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status_label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="dc_voltage_label">
       <property name="text">
        <string>0.00V</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Voltage Level Read</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="readVoltage_pushButton">
       <property name="text">
        <string>Read Voltage</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
